webpack.config.js

```diff
@@ -32,7 +32,8 @@ Encore
         }]
     })
     .enableSingleRuntimeChunk()
-    .enableSourceMaps(!Encore.isProduction())
+    //.enableSourceMaps(!Encore.isProduction())
+    .enableSourceMaps(true)
```

Comment:
Not sure, since its unrelated, this is a 'leftover' or you changed it on purpose?

Reply:
Nice find. It's a testing-leftover. I've removed it